src/content/docs/fr/index.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Base renforcée pour vos applications web.
+description: Les outils de création d'applications multiplatformes
+i18nReady: true
+template: splash
+hero:
+  image:
+    file: ../../../assets/logo-outline.svg
+  actions:
+    - text: Commencer
+      link: fr/2/guide/
+      icon: right-arrow
+      variant: primary
+    - text: Directives de Contrubution
+      link: fr/contribute/
+      icon: open-book
+      variant: minimal
+    - text: Documentation de Tauri 1.0
+      link: https://tauri.app
+      icon: external
+      variant: minimal
+banner:
+  content: |
+    Vous êtes sur l'avant-première du site pour Tauri 2.0 -
+    <a href="https://tauri.app">Aller au site de Tauri 1.0</a>
+---
+
+import { Card, CardGrid } from '@astrojs/starlight/components';
+
+<div class="hero-bg">
+	<div class="bg-logo"></div>
+	<div class="bg-grad"></div>
+	<div class="bg-grad-red"></div>
+</div>
+
+<CardGrid stagger>
+	<Card title="Frontend Indépendant" icon="rocket">
+		Importez votre configuration web existant dans Tauri ou démarrez ce nouveau
+		projet de rêve.
+		Tauri supporte n'importe quel framework de frontend, vous n'avez donc
+		pas besoin de changer votre configuration.
+	</Card>
+	<Card title="Multiplatforme" icon="rocket">
+		Développez vos applications pour Linux, macOS, Windows, Android et iOS
+		- tout cela à partir du même code.

```

Comment:
```suggestion
		Développez vos applications pour Linux, macOS, Windows, Android et iOS - tout cela à partir du même code.

```

Looks like this was being treated as a bullet, so just needs to be put on the same line

Reply:
oh yeah didn't thought about it